core-bundle/src/Twig/Interop/PhpTemplateProxyNode.php

```diff
@@ -49,7 +55,18 @@ public function compile(Compiler $compiler): void
             ->write('function(callable $block) use ($context): string {'."\n")
             ->indent()
             ->write('if ($this->env->isDebug()) { ob_start(); } else { ob_start(static function () { return \'\'; }); }'."\n")
-            ->write('try { $block($context); return ob_get_contents(); } finally { ob_end_clean(); }'."\n")
+            ->write('try {'."\n")
+            ->indent()
+            ->write('$content = \'\';'."\n")
+            ->write('foreach ($block($context) ?? [\'\'] as $chunk) {'."\n")
+            ->indent()
+            ->write('$content .= ob_get_contents() . $chunk;'."\n")
+            ->write('ob_clean();'."\n")
+            ->outdent()
+            ->write('}'."\n")
+            ->write('return $content . ob_get_contents();'."\n")
+            ->outdent()
+            ->write('} finally { ob_end_clean(); }'."\n")
```

Comment:
We should update the comment in line 41 accordingly, shouldn‘t we?

Reply:
+1 for dropping them. 
Maybe replace them with a `/** @see PhpTemplateProxyNodeTest::testCompilesProxyCode() */` ?

Reply:
Perfect. 👍

Reply:
Changed in 703fb6c1f0252f963f41fba9a36cc60229453c35.